features/dd-sdk-android-trace/src/test/kotlin/com/datadog/android/utils/forge/CoreDDSpanForgeryFactory.kt

```diff
@@ -29,6 +29,7 @@ internal class CoreDDSpanForgeryFactory : ForgeryFactory<DDSpan> {
         val traceId = forge.aLong(min = 1)
         val spanId = forge.aLong(min = 1)
         val parentId = forge.aLong(min = 1)
+        val samplingPriority = forge.anInt()
```

Comment:
should we generate the value from the 4 values we can have? or it doesn't matter here?

Reply:
it doesn't matter here it is ok to be a random one.